from chat.exceptions.not_provided_exception import (
    NotProvidedException as NotProvidedException
)